### PR TITLE
Replace multi-service ironic container with several containers

### PR DESCRIPTION
**Pull request by dtantsur**
Requires https://github.com/metal3-io/ironic-image/pull/62

### CODE REVIEW

File: 06_create_cluster.sh [hunk 1]
```diff
@@ -60,7 +60,7 @@ wait_for_json ironic \
     20 \
     -H "Accept: application/json" -H "Content-Type: application/json" -H "User-Agent: wait-for-json" -H "X-Auth-Token: $OS_TOKEN"
 
-if [ $(sudo podman ps | grep -w -e "ironic$" -e "ironic-inspector$" -e "dnsmasq" -e "httpd" | wc -l) != 4 ]; then
+if [ $(sudo podman ps | grep -w -e "ironic-api$" -e "ironic-conductor$" -e "ironic-inspector$" -e "dnsmasq" -e "httpd" | wc -l) != 5 ]; then
```
Comment (bfournie) [inline 06_create_cluster.sh]: Should this check ironic-exporter too?

Comment (dtantsur) [reply 06_create_cluster.sh]: It's not strictly required for deployment, so I'd avoid it here.

**Comment by hardys** [comment]
Note that pretty soon we'll move away from dev-scripts managing all the ironic things, so we'll need to align https://github.com/metal3-io/baremetal-operator/pull/212 https://github.com/openshift-metal3/dev-scripts/pull/635 and https://github.com/openshift-metal3/kni-installer/pull/100 with this.

**Comment by dtantsur** [comment]
Right. This patch is still useful to prove that the split works (I'll run the CI when the dependency merges).

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/842/

**Comment by hardys** [comment]
Ok this passed CI and we'll need this landed to test a follow-up PR to ironic-image that adds logging for the new entrypoints, so lets go ahead and merge this.

**Pull request merged**